node-tests/htmlbars-minifier.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
This tests are fine but you should migrate the `node-tests` to check that the "plugin" is well constructed.

I know this is out of the scope of this task :S Check [this example](https://github.com/BBVAEngineering/broccoli-thumbnail/blob/master/tests/thumbnail-test.js) about using the Broccoli helper.

Reply:
Yes, is the next goal we've planned to reach :) Thanks!